spec/features/schools/ects/change_email_address_spec.rb

```diff
@@ -0,0 +1,125 @@
+describe "School user can change ECTs email address" do
+  before do
+    allow(Rails.application.config)
+      .to receive(:enable_schools_interface)
+      .and_return(true)
+  end
+
+  it "changes the email address" do
+    given_there_is_a_school
+    and_there_is_an_ect
+    and_i_am_logged_in_as_a_school_user
+
+    when_i_visit_the_ect_page
+    then_i_can_change_the_email_address
+    and_i_see_the_change_email_form
+
+    when_i_do_not_enter_an_email_address
+    and_i_continue
+    then_i_see_an_error("Enter an email address")
+
+    when_i_do_not_enter_a_different_email_address
+    and_i_continue
+    then_i_see_an_error("The email must be different from the current email")
+
+    when_i_enter_an_invalid_email_address
+    and_i_continue
+    then_i_see_an_error("Enter an email address in the correct format, like name@example.com")
+
+    when_i_enter_an_email_address_that_is_too_long
+    and_i_continue
+    then_i_see_an_error("Enter an email address that is less than 254 characters long")
```

Comment:
Our specs are getting slower and slower every day, could we create step specs and test the validations in there please? It should help keep our feature tests focused and faster 🙏🏽

Reply:
@cpjmcquillan you might be able to copy the ones for "change name" or turn them into shared bits-n-bobs.

Reply:
I've added some unit tests for the steps.

Personally I think feature tests that touch the unhappy paths are useful, but I've removed them 👌🏼